src/SFA.DAS.ProviderApprenticeshipsService.Infrastructure/Services/StubProviderRelationshipsClient.cs

```diff
@@ -17,12 +17,12 @@ public StubProviderRelationshipsApiClient()
             {
                 new KeyValuePair<long, AccountProviderLegalEntityDto>(10005077, new AccountProviderLegalEntityDto
                 {
-                    AccountId = 15623,
-                    AccountPublicHashedId = "74449P",
-                    AccountLegalEntityName = "RED DIAMONDS",
-                    AccountLegalEntityId = 8004,
-                    AccountLegalEntityPublicHashedId = "XPBMMX",
-                    AccountName = "RED DIAMONDS (ACCOUNT)"
+                    AccountId = 8194,
+                    AccountPublicHashedId = "9V6JWR",
+                    AccountLegalEntityName = "ASAP CATERING LIMITED (Stub)",
+                    AccountLegalEntityId = 3884,
+                    AccountLegalEntityPublicHashedId = "94DVK9",
+                    AccountName = "ASAP CATERING LIMITED (Stub) (ACCOUNT)"
```

Comment:
Can we set this up to be a valid entry in EmployerAccounts DB in the AT environment as well as in your stub implementation. This is just because the Config we get from the SFA-DAS-employer-configs repo points to the AT world

Or we could update the config to point to a stub (but that means setting up Node)